src/coordinator/storage/block.go

```diff
@@ -144,14 +156,19 @@ func (m *multiSeriesBlockSeriesIter) Next() bool {
 
 func (m *multiSeriesBlockSeriesIter) Current() block.Series {
```

Comment:
Could potentially be better to move this computation to the `Next()` function rather than `Current()`, depending on the usage we expect from this type. If we're calling Current() often, should pre-compute on `Next()` to save unnecessary processing.

Reply:
Next() and Current() are called the same number of times (until Next() returns true) so don't think we can save here.